### PR TITLE
Normalize Ports

### LINKED ISSUE #17: Avoid standard ports
Set ports of components in docker-compose file to non-standard ports (avoid 8080, 80, etc.).
Make sure that not the same ports as for the OpenTOSCA docker-compose are used.

### PR DESCRIPTION
**Pull request by lharzenetter**
To enable a better development expierence, we decided to change the ports of all services such that they do not conflict with other ecosystems like [OpenTOSCA](https://github.com/OpenTOSCA).

Therefore, we agreed to change the ports as follows:

```
    DB:
        5432            --> 5060
    QC Atlas UI
        80
        4200 -- dev     --> 4210
    QC Atlas
        8080            --> 6626
    Pattern Atlas
        8082            --> 1977
        8080 -- dev     ----┘
    Pattern Atals UI
        81
        4200 -- dev     --> 1978
    Nisq Analyzer
        8081            --> 5010
        8080 -- dev     ----┘
    Latex Render
        8083            --> 5030
        8082 -- dev     ----┘
    Quantum transpiler
        5000            --> 5012
    Quantum transpiler UI
        8086            --> 5011
        4200 -- dev     --> 4212
    Qiskit service
        5003            --> 5013
        5000 -- dev
    forest-service
        5002            --> 5014
        5000 -- dev     ----┘
    pytket service
        5001            --> 5015
        5000 -- dev     ----┘
    rigetti qvm
        5666            --> 5016
    rigetti quilic
        5667            --> 5017
    redis
        6379            --> 5040
    qprov
        8084            --> 5020
        8080 -- dev     ----┘
    qprov collector ibm
        8085            --> 5021
        8081 -- dev     ----┘
    config service
        2379
```

### CODE REVIEW

**Comment by lharzenetter** [comment]
https://github.com/UST-QuAntiL/qc-atlas-ui/pull/137

File: docker-compose.yml [hunk 1]
```diff
@@ -87,40 +87,40 @@ services:
...
-      - '8081:8080'
+      - '5010:5010'
     networks:
       - default
   # used by qc-atlas-ui to render latex
   latex-renderer:
     image: planqk/latex-renderer:v1.0.1
     ports:
-      - "8083:8082"
+      - "5012:5012"
```
Comment (wederbn) [inline docker-compose.yml]: I think it should be 5030?

File: docker-compose.yml [hunk 2]
```diff
@@ -87,40 +87,40 @@ services:
...
 
   quantum-transpiler:
     image: planqk/quantum-transpiler-backend:v1.0.0
     profiles: ["all", "nisqAnalyzerCompilerComparison"]
     ports:
-     - '5000:5000'
+     - '5011:5011'
```
Comment (wederbn) [inline docker-compose.yml]: The backend should be 5012?

File: docker-compose.yml [hunk 3]
```diff
@@ -158,14 +158,14 @@ services:
     profiles: ["all", "nisqAnalyzerCompilerComparison"]
     command: sh /app/startup.sh
     ports:
-      - "5002:5000"
+      - "5014:5014"
```
Comment (wederbn) [inline docker-compose.yml:161]: The change for the Pytket service (5015) is missing?

**Review by wederbn — COMMENTED** [review]
Maybe I'm a bit confused by all the numbers but please double-check the three comments I added :-)

**Comment by lharzenetter** [comment]
https://github.com/PatternAtlas/pattern-atlas-docker/pull/4

**Comment by lharzenetter** [comment]
https://github.com/UST-QuAntiL/latex-renderer/pull/7

**Comment by lharzenetter** [comment]
https://github.com/PatternAtlas/pattern-atlas-ui/pull/109

**Review by wederbn — APPROVED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -41,33 +41,35 @@ For running all feature sets, choose `--profile all`.
...
-| Quantum-Transpiler-Backend |<http://localhost:5790> | [Link](https://github.com/UST-QuAntiL/QuantumTranspiler) | [Link](https://hub.docker.com/r/planqk/quantum-transpiler-backend) |
-| Quantum-Transpiler-Frontend |<http://localhost:8086> | [Link](https://github.com/UST-QuAntiL/QuantumTranspiler) | [Link](https://hub.docker.com/r/planqk/quantum-transpiler-frontend) |
-| Redis |<http://localhost:6379> | [Link](https://github.com/redis/redis) | [Link](https://hub.docker.com/_/redis) |
-| Rigetti QVM |<http://localhost:5666> | [Link](https://github.com/rigetti/qvm) | [Link](https://hub.docker.com/r/rigetti/qvm) |
-| Rigetti Quilc |<http://localhost:5667> | [Link](https://github.com/rigetti/quilc) | [Link](https://hub.docker.com/r/rigetti/quilc) |
-| QProv |<http://localhost:8084/qprov> | [Link](https://github.com/UST-QuAntiL/qprov) | [Link](https://hub.docker.com/r/planqk/qprov) |
-| QProv-Collector-IBM |<http://localhost:8085> | [Link](https://github.com/UST-QuAntiL/qprov) | [Link](https://hub.docker.com/r/planqk/qprov-collector) |
+| QC-Atlas-UI |<http://localhost> | [Link](https://github.com/UST-QuAntiL/qc-atlas-ui) | [Link](https://hub.docker.com/r/planqk/qc-atlas-ui) |
```
Comment (salmma) [inline README.md]: port number is missing at localhost

File: docker-compose.yml [hunk 5]
```diff
@@ -179,9 +179,9 @@ services:
       - REDIS_URL=redis://redis
       - DATABASE_URL=sqlite:////data/app.db
       - QVM_HOSTNAME=rigetti-qvm
-      - QVM_PORT=5666
+      - QVM_PORT=5016
```
Comment (salmma) [inline docker-compose.yml:182]: please adapt QVM_PORT=5016 at line 216 for pytket-worker

Comment (salmma) [reply docker-compose.yml:182]: and also for pytket-service in line 202

**Review by salmma — CHANGES_REQUESTED** [review]
please adapt the requested changes :)

File: README.md (hunk 4 above)
Comment (lharzenetter) [reply README.md]: It is not necessary since 80 is the default port for http. However, I could add it again if you want to maintain consistency?

Comment (salmma) [reply README.md]: I like consistency :D

**Comment by lharzenetter** [comment]
TODO: Update container version tags

**Review by salmma — APPROVED** [review]

**Pull request merged**